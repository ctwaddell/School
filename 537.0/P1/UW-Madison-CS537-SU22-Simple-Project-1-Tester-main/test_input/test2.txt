abandon
ABANDON
a'bandon
a'[].,bandon
band
aBaNdOn

abuse
amazing

