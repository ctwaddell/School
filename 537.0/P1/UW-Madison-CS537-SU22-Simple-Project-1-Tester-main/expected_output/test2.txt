abandon
ABANDON
a'bandon
a'[].,bandon
aBaNdOn
